src/WorkflowDefinition.php

```diff
@@ -59,7 +59,9 @@ public function addJob(
         }
 
         $this->jobs[$id] = [
-            'job' => $job,
+            'job' => $job
+                ->withJobId($id)
+                ->withStepId(Str::orderedUuid()),
```

Comment:
I don't quite understand this change. Does calling these two methods here instead of the `build` method change anything?

Reply:
On top of my mind, I think it was for [WorkflowStep change](https://github.com/ksassnowski/venture/pull/30/files#diff-8798444bdb96c6c1a87aab9e4b97d81245d2465eab26ab8e01e8da27300db5c9R36) when nesting workflows. In that case, the build method havent provided the jobs with any identifier that can be used while nesting. A few tests fail when these id's are moved to their original place.